.github/workflows/deploy.yml

```diff
@@ -52,5 +52,19 @@ jobs: # We need two separate builds as each is going to a different ECR repo
     secrets:
       aws_account_id: ${{ secrets.AWS_ACCOUNT_ID }}
       aws_role_id: ${{ secrets.AWS_ROLE_ID }}
-      aws_region: ${{ secrets.AWS_REGION }}
-      aws_ecr_repository: ${{ secrets.AWS_ECR_REPOSITORY }}
+      aws_region: ${{ secrets.AWS_REGION_PROD_0 }}
+      aws_ecr_repository: ${{ secrets.AWS_ECR_REPOSITORY_PROD_0 }}
+  deploy-to-prod-1:
+    uses: ipfs-elastic-provider/shared-workflows/.github/workflows/lambda-shared-deploy.yaml@main
+    permissions:
+      id-token: write
+    needs: build-prod
+    with:
+      deploy_to_environment: prod
+      lambda: side-indexer
+      docker_image_version: ${{ needs.build-prod.outputs.build_date }}
+    secrets:
+      aws_account_id: ${{ secrets.AWS_ACCOUNT_ID }}
+      aws_role_id: ${{ secrets.AWS_ROLE_ID }}
+      aws_region: ${{ secrets.AWS_REGION_PROD_1 }}
+      aws_ecr_repository: ${{ secrets.AWS_ECR_REPOSITORY_PROD_1 }}
```

Comment:
Nothing was pushed to this ECR repo before, so no updates will be made.
I'll fix it.